Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -433,11 +433,29 @@ public static RHG.Brep ToRhino(this List<BHG.ISurface> surfaces)
 
         /***************************************************/
 
-        [NotImplemented]
         public static RHG.Extrusion ToRhino(this BHG.Extrusion extrusion)
         {
-            // TODO Rhino_Adapter conversion to Extrusion
-            return null;
+            if (!extrusion.Curve.IIsPlanar()) {
+                BH.Engine.Reflection.Compute.RecordError("The provided BHoM Extrusion has a base curve that is not planar.");
+                return null;
+            }
+
+            var planarCurve = extrusion.Curve.IToRhino();
+
+            RHG.Plane curvePlane;
+            planarCurve.TryGetPlane(out curvePlane);
+
+
+            double extrHeight = extrusion.Direction.Length();
+
+            double angle = RHG.Vector3d.VectorAngle(curvePlane.Normal, extrusion.Direction.ToRhino()) * (180 / Math.PI);
+
+            if (angle > 90)
+                extrHeight = -extrHeight; 
+
+            RHG.Extrusion extr = Rhino.Geometry.Extrusion.Create(planarCurve, extrHeight, extrusion.Capped);
+
+            return extr;
         }
 
```

Comment:
Just having a quick look at this. Need to bear in mind that `BH.oM.Geometry.Extrusion` does not enforce extrusions perpendicular to the curve. So this outcome below possible with the current code. It is not clear to me if a BHoM extrusion is / was supposed to be constrained to only extrude normal to the plane of the curve.
![Screenshot 2021-02-23 092022](https://user-images.githubusercontent.com/6618854/108823674-a8b81280-75b8-11eb-988f-07e4c3ef4c11.jpg)

Reply:
Thanks @rolyhudson . See commit https://github.com/BHoM/Rhinoceros_Toolkit/pull/204/commits/d24289bd6b966d4ac4e988c3fb8c079c8a04e2a5